reduce brands/brand confusion on Brands API docs (#3344)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

## Get info about brands and themes

At the top level, Your Okta org contains a single brand, which contains a default theme. The brand is applied to your org's subdomain/[custom domain](/docs/guides/custom-url-domain/) if you have specified one.

> **Important:** Despite being called the Brands API (due to conventions around REST API naming), each org can currently contain only one brand and one theme. We will likely allow multiple brands and themes per org at some point in the future, so stay tuned!

### Get brands

> You can return the org's brands with the following request (**Get brands** in Postman):

Reviewer: What do you think of this?
```suggestion
You can return the org's brand response objects with the following request (**Get brands** in Postman):
```

<ApiOperation method="get" url="/api/v1/brands" />

> This returns an array of [brand response objects](/docs/reference/api/brands/#brand-response-object), which will currently contain one single object only. You can try this in Postman by running the Get brands request.

Reviewer: ```suggestion
This returns an array of [brand response objects](/docs/reference/api/brands/#brand-response-object), which currently contains a single object. You can try this in Postman by running the GET brands request.
```

<ApiOperation method="get" url="/api/v1/brands/${brandId}/themes" />

> This returns an array of [theme response objects](/docs/reference/api/brands/#theme-response-object), which will currently contain one single object only.

Reviewer: ```suggestion
This returns an array of [theme response objects](/docs/reference/api/brands/#theme-response-object), which currently contain a single object.
```
